indico/modules/categories/templates/display/event_list.html

```diff
@@ -102,6 +102,14 @@ <h4 class="{% if month.is_current %}current-month{% endif %}">
                 <li>
                     <span class="ical">
                         <a class="icon-calendar" href="{{ url_for('events.export_event_ical', event) }}"></a>
+                        {% if session.user and event in session.user.favorite_events %}
+                            {% set tooltip %}
+                                {%- trans -%}
+                                    You have favorited this event.
+                                {%- endtrans -%}
+                            {% endset %}
+                            <i class="icon-star" title="{{ tooltip }}"></i>
```

Comment:
```suggestion
                            <i class="icon-star" title="{% trans %}You have favorited this event.{% endtrans %}"></i>
```

still fits nicely in one line here ;)